src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -301,6 +303,22 @@ private ContextMenu updateContextMenu(ContextMenu contextMenu) {
         return contextMenu;
     }
 
+    private ObservableList<TurboIssue> applyCountQualifierToIssueList(ObservableList<TurboIssue> issueList,
```

Comment:
Yup, looked at `sort`. I have added two methods in `UpdateController` that do the job of applying the count filter. `sort` is given preference over `count`.